peekingduck/pipeline/nodes/draw/bbox.py

```diff
@@ -40,7 +40,8 @@ class Node(AbstractNode):
         |none|
 
     Configs:
-        None.
+        show_labels (:obj:`bool`): **default = False**
+            whether to show bbox class labels name.
```

Comment:
```suggestion
            Show class label, e.g. "person", above bounding box
```
This is more consistent with others I think:
https://peekingduck.readthedocs.io/en/stable/peekingduck.pipeline.nodes.dabble.fps.Node.html#peekingduck.pipeline.nodes.dabble.fps.Node